pkg/controller/intrusiondetection/intrusiondetection_controller.go

```diff
@@ -19,6 +19,12 @@ import (
 	"fmt"
 	"time"
 
+	"k8s.io/apimachinery/pkg/api/resource"
```

Comment:
Can you please organise these imports in the same manner as here https://github.com/tigera/operator/blob/271d1b851db73b758c6c986f7f2f5170e253b724/pkg/controller/installation/core_controller.go#L17-L75.